playbooks/conferences.md

```diff
@@ -24,6 +24,30 @@ guidelines:
 - If you need help preparing a conference talk, stop by [Ash][ash]'s weekly Writing Office Hours.
 - It's also recommended sharing the talk and lessons with the team.
 
+## Approval Process
+
+- Find an event (conference/seminar/course) that you would like to attend. Identify items in the agenda that will
+  contribute to both your growth goals and Artsy's business goals.
+- Estimate any travel or lodging expenses that may be needed to attend the event using Concur to search for airfare
+  and hotels.
```

Comment:
Should we include some links to flight aggregators or at least mention that you have the option of buying flights outside of Concur and expensing them?

In my case, I was able to find a shorter itinerary with less expensive fares outside of concur. I also booked an AirBnB instead of booking a hotel through Concur.

Reply:
We have an airbnb business account we can use for booking one. It could be useful to link back to [Atlas](https://sites.google.com/a/artsymail.com/intranet/travel-accommodations) for useful resources, for example, the [rate card](https://docs.google.com/spreadsheets/d/1nFR-tgxqqct-RjYhcqxKhUKLHxZLhWMKImHaItt_XV0/edit#gid=0).

Reply:
Ohh saw you included the link below! 👍